### PR TITLE
Add a button for adding accounts

### PR DESCRIPTION
**Pull request by ivanhrabcak**

### CODE REVIEW

File: app/app.iml [hunk 1]
```diff
@@ -30,15 +30,15 @@
     <output-test url="file://$MODULE_DIR$/build/intermediates/javac/debugUnitTest/classes" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/build/generated/ap_generated_sources/debug/out" isTestSource="false" generated="true" />
-      <sourceFolder url="file://$MODULE_DIR$/build/generated/aidl_source_output_dir/debug/out" isTestSource="false" generated="true" />
-      <sourceFolder url="file://$MODULE_DIR$/build/generated/source/buildConfig/debug" isTestSource="false" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/build/generated/renderscript_source_output_dir/debug/out" isTestSource="false" generated="true" />
+      <sourceFolder url="file://$MODULE_DIR$/build/generated/source/buildConfig/debug" isTestSource="false" generated="true" />
```
Comment (gabrielKerekes) [inline app/app.iml:34]: Subor by mal byt v .gitignore. Pozri toto:

https://github.com/github/gitignore/blob/master/Android.gitignore

File: app/src/main/java/com/ivik/passwordmanager/MainActivity.java [hunk 2]
```diff
@@ -22,18 +27,79 @@
...
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    private void addNewAccount() {
+        final AlertDialog.Builder builder = new AlertDialog.Builder(this);
```
Comment (gabrielKerekes) [inline app/src/main/java/com/ivik/passwordmanager/MainActivity.java:51]: Bolo by pekne keby toto vies vybrat do classy AddNewAccountDialog a tam dat celu tuto metodku.

File: app/src/main/res/drawable/plus_button.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (gabrielKerekes) [inline app/src/main/res/drawable/plus_button.xml:1]: Zaujimave riesenie ;) Ja som vzdy hladal pngcka na nete. Nice!

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -3,4 +3,8 @@
     <string name="accounts">Accounts</string>
     <string name="username">Username:</string>
     <string name="password">Password:</string>
+    <string name="password_input">Password</string>
```
Comment (gabrielKerekes) [inline app/src/main/res/values/strings.xml:6]: Super, ze to pouzivas! Ja som na to dlho kaslal. Je dobre si na to zvyknut :)

File: app/src/main/res/menu/menu.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <item
+        android:id="@+id/account_add_button"
+        android:icon="@drawable/plus_button"
+        app:showAsAction="always"
+        android:title="Add account">
```
Comment (gabrielKerekes) [inline app/src/main/res/menu/menu.xml:11]: Tuto vies tiez pouzit stringy zo `strings.xml`

File: app/src/main/java/com/ivik/passwordmanager/MainActivity.java [hunk 6]
```diff
@@ -22,18 +27,79 @@
...
+        builder.setTitle("Add a new account:");
+
+        LinearLayout linearLayout = new LinearLayout(this);
```
Comment (gabrielKerekes) [inline app/src/main/java/com/ivik/passwordmanager/MainActivity.java:54]: Dialogy tiez vedia mat .xml layouty, cize nemusis ho tu programaticky vytvarat. Je to tak lepsie. Pozri [tu](https://developer.android.com/guide/topics/ui/dialogs#CustomLayout).

**Review by gabrielKerekes — COMMENTED** [review]
Nespustal som si to. Ale asi si to nerobil s [FAB](https://developer.android.com/guide/topics/ui/floating-action-button). Bolo by pekne keby si to skusis s tymto buttonom, ale ako chces :) Ak to ale funguje, tak aj tak good job  :)

File: app/src/main/java/com/ivik/passwordmanager/MainActivity.java (hunk 6 above)
Comment (ivanhrabcak) [reply app/src/main/java/com/ivik/passwordmanager/MainActivity.java:54]: Skusal som to, ale nevedel som tie EditText-y najst cez findViewById.

Comment (ivanhrabcak) [reply app/src/main/java/com/ivik/passwordmanager/MainActivity.java:54]: Vyskusam znova.

File: app/src/main/res/drawable/plus_button.xml (hunk 3 above)
Comment (ivanhrabcak) [reply app/src/main/res/drawable/plus_button.xml:1]: https://materialdesignicons.com/
Da sa to odtial zkopirovat ako drawable aj ako png

File: app/app.iml (hunk 1 above)
Comment (ivanhrabcak) [reply app/app.iml:34]: [f36c6a](https://github.com/ivanhrabcak/PasswordManager/commit/f36c6a93800be278030126391fa8d4d8300885f3) - add android files to gitignore

**Pull request merged**